core-bundle/src/Picker/ArticlePickerProvider.php

```diff
@@ -84,8 +109,12 @@ public function getDcaAttributes(PickerConfig $config): array
     /**
      * {@inheritdoc}
      */
-    public function convertDcaValue(PickerConfig $config, $value): string
+    public function convertDcaValue(PickerConfig $config, $value)
```

Comment:
Why did you remove the return type?

Reply:
because it can be an integer as well now